pkg/storaged/client.js

```diff
@@ -621,6 +623,35 @@
         return Promise.resolve();
 };
 
+/* Direct mounting and unmounting
+ *
+ * We don't use UDisks2 for most of our mounting and unmounting in
+ * order to get better control over which entry from fstab is
+ * selected.  Once UDisks2 allows that control, we can switch back to
+ * it.
+ *
+ * But note that these functions still require there to be an entry in
+ * fstab.
+ */
+
+client.mount_at = (block, target) => {
+    const entry = utils.array_find(block.Configuration,
+                                   c => c[0] == "fstab" && utils.decode_filename(c[1].dir.v) == target);
+    if (entry)
+        return cockpit.script('set -e; mkdir -p "$2"; mount "$1" "$2" -o "$3"',
+                              [utils.decode_filename(block.Device), target, utils.decode_filename(entry[1].opts.v)],
+                              { superuser: true, err: "message" });
+    else
+        return Promise.reject(cockpit.format(_("No fstab entry for $0 and $1"),
+                                             utils.decode_filename(block.Device),
+                                             target));
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16991-20220830-101236-2e5e4dfa-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Reaching this is more a programming error than a user error, so it's hard to test.

Reply:
In that case, I'm in favor of dropping the gettext() -- no need to waste translator efforts on something users should never see?

Reply:
Yes, I'll also add "Internal error:".